ml/src/test/scala/ai/h2o/sparkling/ml/models/H2OMOJOPipelineModelTestSuite.scala

```diff
@@ -314,4 +291,27 @@ class H2OMOJOPipelineModelTestSuite extends FunSuite with SparkTestContext with
     val settings = H2OMOJOSettings()
     testTransformAndTransformSchemaAreAligned(settings)
   }
+
+  test("Mojo pipeline can expose contribution (SHAP) values") {
+
+    val mojoSettings = H2OMOJOSettings(withContributions = true)
+    val pipeline = H2OMOJOPipelineModel.createFromMojo(
+      this.getClass.getClassLoader.getResourceAsStream("daiMojoShapley/pipeline.mojo"),
+      "pipeline.mojo",
+      mojoSettings)
+
+    val df = spark.read.option("header", "true").csv("ml/src/test/resources/daiMojoShapley/example.csv")
+    val transformed = pipeline.transform(df)
+    val predictionsAndContributions = transformed.select("prediction.*")
```

Comment:
Can we separate predictions and contributions? (one composed column with predictions and another composed column named `contributions`)

Reply:
I modified the code to produce a separate **contribution** column. I didn't use the plural here as predictions are provided in **prediction** column (singular form) so I wanted to stay consistent.